lib/Chat/CommentsManager.php

```diff
@@ -39,4 +40,109 @@ public function getCommentFromData(array $data): IComment {
 		$comment->setMessage($message, ChatManager::MAX_CHAT_LENGTH);
 		return $comment;
 	}
+
+	/**
+	 * TODO Remove when moved to server
+	 *
+	 * @param string $objectType the object type, e.g. 'files'
+	 * @param string $objectId the id of the object
+	 * @param string[] $verbs List of verbs to filter by
+	 * @param int $lastKnownCommentId the last known comment (will be used as offset)
+	 * @param string $sortDirection direction of the comments (`asc` or `desc`)
+	 * @param int $limit optional, number of maximum comments to be returned. if
+	 * set to 0, all comments are returned.
+	 * @param bool $includeLastKnown
+	 * @return IComment[]
+	 * @return array
+	 */
+	public function getCommentsWithVerbForObjectSinceComment(
+		string $objectType,
+		string $objectId,
+		array $verbs,
+		int $lastKnownCommentId,
+		string $sortDirection = 'asc',
```

Comment:
Is necessary? The unique place that `getCommentsWithVerbForObjectSinceComment` was used force the sort to `'desc'`.

Reply:
See comment above:
> Method is a duplicate of getForObjectSince with the verbs parameter added. Will deduplicate (getForObjectSince will call getCommentsWithVerbForObjectSinceComment with empty array) in server.

So this is going to replace the exist method and needs to support all previous parameters

Reply:
PR in https://github.com/nextcloud/server/pull/30794